state-chain/chains/src/btc/smart_contract_encoding.rs

```diff
@@ -0,0 +1,95 @@
+use crate::ForeignChainAddress;
+use cf_primitives::{Asset, AssetAmount};
+use codec::{Decode, Encode};
+
+use super::{BitcoinOp, BitcoinScript};
+
+// The maximum length of data that can be encoded in a nulldata utxo
+const MAX_NULLDATA_UTXO_LENGTH: usize = 80;
```

Comment:
```suggestion
const MAX_NULLDATA_LENGTH: usize = 80;
```

Strictly speaking the utxo is longer than that (because it includes the opcodes).

Reply:
Yeah, I thought this was confusing, but didn't think we could do better without making the name even longer. `MAX_NULLDATA_LENGTH` also isn't great, as to me it sounds like we are talking about an array of 80 zeros, rather than nulldata being simply the type of the utxo... No strong opinion though, I updated to use your suggestion.